app/models/course/settings/survey_component.rb

```diff
@@ -9,6 +9,29 @@ def self.email_setting_items
     }
   end
 
+  def lesson_plan_item_settings
+    enabled_setting = settings.settings(:lesson_plan_items).enabled
+    visible_setting = settings.settings(:lesson_plan_items).visible
+    {
+      component: key,
+      component_title: I18n.t('components.surveys.name'),
+      enabled: enabled_setting.nil? ? true : enabled_setting,
```

Comment:
Forgot to mention earlier, but should setting the default value be the responsibility of `Course::Settings::LessonPlanItems`?

Reply:
No I think each component should be able to declare its default.